content/en/logs/log_collection/nodejs.md

```diff
@@ -132,12 +128,24 @@ logs:
     sourcecategory: sourcecode
 ```
 
+3. [Restart the Agent][9].
+4. Run the [Agent’s status subcommand][10] and look for `java` under the `Checks` section to confirm logs are successfully submitted to Datadog.
```

Comment:
Should this by `nodejs` or something similar instead of `java`?

Reply:
Yes, thanks for catching that!